src/microbe_stage/ProcessSystem.cs

```diff
@@ -207,6 +212,77 @@ void MakeSureResultExists(Compound compound)
         return ComputeCompoundBalance(organelles.Select(o => o.Definition), biome, amountType);
     }
 
+    /// <summary>
+    ///   Computes the compound balances for given organelle list in a patch and at a given time during the day (or
+    ///   using longer timespan values)
+    /// </summary>
+    /// <remarks>
+    ///   <para>
+    ///     Assumes that the cell produces at most as much ATP as it consumes
+    ///   </para>
+    /// </remarks>
+    public static Dictionary<Compound, CompoundBalance> ComputeCompoundBalanceAtEquilibrium(
+        IEnumerable<OrganelleTemplate> organelles, BiomeConditions biome, CompoundAmountType amountType,
```

Comment:
Is there a reason why this takes in organelle templates and not organelle definitions like the `ComputeCompoundBalance` method?

Reply:
Not really actually, I can change this to be consistent with the other method.